modules/microshift-custom-ca-con.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-custom-cas_{context}"]
+= How custom certificate authorities work in {microshift-short}
+
+The default API server certificate is issued by an internal {microshift-short} cluster certificate authority (CA). Clients outside of the cluster cannot verify the API server certificate by default. This certificate can be replaced by a custom server certificate that is issued externally by a custom CA that clients trust. The following steps illustrate the workflow in {microshift-short}:
+
+. Set up your CAs. Updating the certificate bundle with the configuration that contains the CA is recommended. If you do not want to configure your certificate bundles, you can alternately use the `oc login localhost:8443 --certificate-authority=/path/to/cert.crt` command, but this method is more manual and not preferred.
+
```

Comment:
before configuring the custom certs in `/etc/microshift/config.yaml`, there should be a step that instructs to copy the certificate content + key to the Microshift host [see](https://github.com/openshift/enhancements/blob/master/enhancements/microshift/microshift-apiserver-custom-certs.md#when-custom-namedcertificates-configured). (also worth mentioning that the files should be accesible to root only)